packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/LabelType.java

```diff
@@ -22,6 +22,15 @@ public enum LabelType {
     OUTSIDE("label:outside"),
     INSIDE_CENTER("label:inside-center"),
     OUTSIDE_CENTER("label:outside-center"),
+    INSIDE_V_TOP_H_CENTER("label:inside-v_top-h_center"),
+    INSIDE_V_TOP_H_LEFT("label:inside-v_top-h_left"),
+    INSIDE_V_TOP_H_RIGHT("label:inside-v_top-h_right"),
+    INSIDE_V_CENTER_H_CENTER("label:inside-v_center-h_center"),
+    INSIDE_V_CENTER_H_LEFT("label:inside-v_center-h_left"),
+    INSIDE_V_CENTER_H_RIGHT("label:inside-v_center-h_right"),
+    INSIDE_V_BOTTOM_H_CENTER("label:inside-v_bottom-h_center"),
+    INSIDE_V_BOTTOM_H_LEFT("label:inside-v_bottom-h_left"),
+    INSIDE_V_BOTTOM_H_RIGHT("label:inside-v_bottom-h_right"),
```

Comment:
What is the difference between `INSIDE_CENTER` and `INSIDE_V_TOP_H_CENTER`? Because right now the default is `INSIDE_CENTER` for everything which is not a border node or an image (in those cases it's outside / outside center) and thus `INSIDE_CENTER` means on top of the node in the middle.

I don't see how `INSIDE_CENTER` can be anything different than one of the `INSIDE_V_{TOP|CENTER|BOTTOM}_H_{LEFT|CENTER|RIGHT}`?

Unless I'm missing something, it should be removed.

Reply:
`INSIDE_CENTER` is used for node parametric svgs, node icon labels, and node rectangles with header.
Then it is used by the frontend in `DependencyInjection.ts`. It allows to make differences between `InsideLabelView.tsx` and `LabelView.tsx`.
And by the way, `INSIDE_CENTER` (a value that exist for a long time) should probably be named `INSIDE` because it is used for every node parametric svgs, node icon labels, and node rectangles with header, no matter the label is at the top, bottom, left, right or center.